addons/binding/org.openhab.binding.modbus/README.md

```diff
@@ -0,0 +1,878 @@
+
+Useful tools
+
+* [binaryconvert.com](http://www.binaryconvert.com/): tool to convert numbers between different binary presentations
+* [rapidscada.net Modbus parser](http://modbus.rapidscada.net/): tool to parse Modbus requests and responses. Useful for debugging purposes when you want to understand the message sent / received.
+* [JSFiddle tool](https://jsfiddle.net/rgypuuxq/) to test JavaScript (JS) transformations interactively
+
+## Supported Things
+
+This binding support 6 different things types
+
+| Thing    | Type   | Description                                                                                                                                                                                                                               |
+| -------- | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `tcp`    | Bridge | Modbus TCP server (Modbus TCP slave)                                                                                                                                                                                                      |
+| `serial` | Bridge | Modbus serial slave                                                                                                                                                                                                                       |
+| `poller` | Bridge | Thing taking care of polling the data from modbus slaves. One poller corresponds to single Modbus read request (FC01, FC02, FC03, or FC04). Is child of `tcp` or `serial`.                                                                |
+| `data`   | Thing  | thing for converting polled data to meaningful numbers. Analogously, is responsible of converting openHAB commands to Modbus write requests. Is child of `poller` (read-only or read-write things) or `tcp`/`serial` (write-only things). |
+
+Typically one defines either `tcp` or `serial` bridge, depending on the variant of Modbus slave.
+For each Modbus read request, a `poller` is defined.
+Finally, one ore more `data` things are introduced to extract relevant numbers from the raw Modbus data.
+For write-only communication, `data` things can be introduced directly as children of `tcp` or `serial` bridges.
+
+## Binding configuration
+
+Other than the things themselves, there is no binding configuration.
+
+## Serial port configuration
+
+If you can see issues related to opening the serial port, and you are using **non standard serial ports** (e.g. `/dev/ttyAMA0`) you might have to configure openHAB to detect and access the port correctly.
+
+Refer to [Serial Port Configuration Notes](http://docs.openhab.org/addons/bindings/serial1/readme.html#port-configuration-notes) for more information.
```

Comment:
Added now various notes regarding serial configuration (some copy-pasted from Serial binding). I guess eventually it would make sense to have this documented in one place -- one place for all bindings to refer.

Reply:
Absolutely agree. So once this PR is merged, maybe you'd be interested in creating a PR in https://github.com/openhab/openhab-docs to add such info at a central place?

Reply:
Certainly! New page directly under "Configuration Guide"?

Reply:
I am not sure about the best place myself - our documentation gods might give you a hint where they think it might fit best once you create a PR.
A new page in the "Administration Guide" might actually be a good fit.